src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCall = 0;
+
+  return (firstNumber) => {
+    countCall++;
+
+    return (secondNumber) => {
+      if (countCall > 3 && countCall % 2 === 0) {
+        return `Bzzz... Error!`;
+      } else {
+        return secondNumber + firstNumber;
+      }
```

Comment:
else statement is redundant
```suggestion
      }
      
      return secondNumber + firstNumber;
```